NuGet.Config

```diff
@@ -54,12 +59,102 @@
+      <package pattern="Microsoft.TeamFoundationServer.ExtendedClient" />
       <package pattern="microsoft.test.apex.visualstudio" />
+      <package pattern="Microsoft.TestPlatform.Portable" />
+      <package pattern="Microsoft.TestPlatform.TestHost" />
       <package pattern="microsoft.visualstudio.*" />
+      <package pattern="Microsoft.VisualStudio.ComponentModelHost" />
+      <package pattern="Microsoft.VisualStudio.LanguageServices" />
+      <package pattern="Microsoft.VisualStudio.ProjectSystem" />
+      <package pattern="Microsoft.VisualStudio.ProjectSystem.Interop" />
+      <package pattern="Microsoft.VisualStudio.ProjectSystem.Managed" />
+      <package pattern="Microsoft.VisualStudio.ProjectSystem.Managed.VS" />
+      <package pattern="Microsoft.VisualStudio.ProjectSystem.VS" />
+      <package pattern="Microsoft.VisualStudio.SDK" />
+      <package pattern="Microsoft.VisualStudio.Sdk.TestFramework.Xunit" />
+      <package pattern="Microsoft.VisualStudio.Setup.Configuration.Interop" />
+      <package pattern="Microsoft.VisualStudio.Shell.15.0" />
+      <package pattern="Microsoft.VisualStudio.VCProjectEngine" />
+      <package pattern="Microsoft.VisualStudio.Workspace.VSIntegration" />
+      <package pattern="Microsoft.VisualStudioEng.MicroBuild.Core" />
+      <package pattern="Microsoft.VSSDK.BuildTools" />
+      <package pattern="Microsoft.Web.Xdt" />
+      <package pattern="Microsoft.Win32.Registry" />
+      <package pattern="Moq" />
+      <package pattern="MSTest.TestAdapter" />
+      <package pattern="MSTest.TestFramework" />
+      <package pattern="Newtonsoft.Json" />
+      <package pattern="NuGet.Client.EndToEnd.TestData" />
+      <package pattern="NuGet.Core" />
+      <package pattern="NuGetValidator" />
+      <package pattern="Portable.BouncyCastle" />
+      <package pattern="SharpZipLib" />
       <package pattern="stdole" />
       <package pattern="streamjsonrpc" />
+      <package pattern="System.Collections" />
+      <package pattern="System.CommandLine" />
+      <package pattern="System.ComponentModel.Composition" />
+      <package pattern="System.Configuration.ConfigurationManager" />
+      <package pattern="System.Diagnostics.Debug" />
+      <package pattern="System.IO.FileSystem.Primitives" />
+      <package pattern="System.Memory" />
+      <package pattern="System.Resources.ResourceManager" />
+      <package pattern="System.Runtime.Extensions" />
+      <package pattern="System.Runtime.InteropServices" />
+      <package pattern="System.Security.Cryptography.Pkcs" />
+      <package pattern="System.Security.Cryptography.ProtectedData" />
+      <package pattern="System.Security.Cryptography.Xml" />
+      <package pattern="System.Text.Encoding.Extensions" />
+      <package pattern="System.Text.Json" />
+      <package pattern="System.Threading" />
+      <package pattern="System.Threading.Tasks" />
+      <package pattern="System.Threading.Tasks.Dataflow" />
```

Comment:
Like other comments, I don't beleive any of these system.* packages, all of which should come fromt he dotnet/runtime repo, have VS specific versions. The system.* mapping that already exists for the dotnet-public feed should restore these successfully.